src/index.html

```diff
@@ -22,20 +22,32 @@ <h3>What is Cassandra?</h3>
         <h4>Proven</h4>
         <p>
         Cassandra is in use at
+        <a href="https://www.dataversity.net/case-study-cassandra-meets-call-of-duty/">Activision</a>,
         <a href="http://www.slideshare.net/daveconnors/cassandra-puppet-scaling-data-at-15-per-month">Constant Contact</a>,
-        <a href="http://planetcassandra.org/blog/post/cassandra-at-cern-large-hadron-collider/">CERN</a>,
+
+        <!-- <a href="http://planetcassandra.org/blog/post/cassandra-at-cern-large-hadron-collider/">CERN</a>, -->
+	<!-- SWITCHED TO SCYLLA https://www.scylladb.com/2018/09/05/cern-real-time-processing-big-data-scylla/ -->
```

Comment:
could we remove the `<!-- … -->` lines? there's no need to keep them, we have them in version control (git).

Reply:
So sorry, had removed and committed, then had to redo.